### PR TITLE
[malwarebytes-254] Changed Domain to threatdown.com

### PR DESCRIPTION
**Pull request by rskumar-mwb**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Malwarebytes/ReleaseNotes/1_1_16.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Malwarebytes
+- Changed Domain malwarebytes.com to threatdown.com.
```
Comment (maimorag) [inline Packs/Malwarebytes/ReleaseNotes/1_1_16.md:3]:
```suggestion

- Updated the Domain from ***malwarebytes.com*** to ***threatdown.com***.

```

**Review by maimorag — COMMENTED** [review]
Hey @rskumar-mwb,
Thank you for your contribution!
Good work :)

Please update the docker image by running:
`demisto-sdk format -ud -i Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.yml`

And add the update to release note by running after committing: 
`demisto-sdk demisto-sdk update-release-notes Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.yml`

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

**Comment by maimorag** [comment]
Hi @rskumar-mwb,
the code looks good!

We're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR.
 Feel free also to send me a recording of a demo.

**Comment by rskumar-mwb** [comment]
Hi @maimorag ,

I have updated the docker image to latest and changed the above mentioned files.

Please see the demo recording below,

https://github.com/demisto/content/assets/48316606/f0fbb131-97bc-4235-9a1c-32ecac498637

The change is only the domain of the api url as our company is changing name soon, the rest of the features should work without any issues as shown in demo recording above.

Let me know if you need me to do any changes.

Thanks.

-RSK

**Review by maimorag — APPROVED** [review]

**Comment by maimorag** [comment]
Hey @rskumar-mwb,
I merged the internal PR, and your update is now in the master branch of content repo.
Your update should be available in the marketplace within the next 72 hours.
Again, thank you very much for your contribution!

**Pull request merged**